flow/controllers/base_controller.py

```diff
@@ -45,6 +49,7 @@ def __init__(self,
                  car_following_params,
                  delay=0,
                  fail_safe=None,
+                 display_warnings=True,
```

Comment:
(will fix this myself). This needs to be added to every child class otherwise it cannot be modified by anything (e.g. IDMController)